book/src/configuration.md

```diff
@@ -2,10 +2,10 @@
 
 To override global configuration parameters, create a `config.toml` file located in your config directory:
 
-* Linux and Mac: `~/.config/helix/config.toml`
-* Windows: `%AppData%\helix\config.toml`
+- Linux and Mac: `~/.config/helix/config.toml`
+- Windows: `%AppData%\helix\config.toml`
```

Comment:
Cheers! I don't use Windows, so this input is important. I don't think I made any changes to this section, @CptPotato has supplied the updated Windows sections, can @CptPotato quickly confirm this change?

Reply:
`%AppData%\helix\config.toml` is actually correct.

`%AppData%` resolves to `C:/Users/<user-name>/AppData/Roaming`.
`%LocalAppData%` resolves to `C:/Users/<user-name>/AppData/Local`.

And I don't think there's an env var for `LocalLow`.

Reply:
Hmmm, I completely forgot about that. Well, there's a case to be made that referring to %AppData% makes it easier for people to find hidden folders. (Unless they're me and have forgotten some of the weird windows obfuscations so now they get lost on things that were supposed to make finding things easier.) On the other hand, I'm using helix to edit my config.toml right now, but the only command prompt under which %AppData% automatically expands for cd is cmd.exe; PowerShell and MinGW are having none of it.

I don't know, but it might help to add a note about the path being an alias for ~/AppData/Roaming/helix/config.toml so that people are more likely to navigate there via a terminal. That should increase the amount of people using helix to edit their helix configs, at least until helix gets a gui version. =)

Reply:
I agree on the bloat issue; bloat itself makes things more confusing as people are trying to take in new information. I dunno, it's a judgment call. If they can find it, they can find it, and either literal path or %AppData% path should point people in the right direction.

As for the powershell thing, nearly every command has a 2-4 letter built-in alias but all of microsoft's own documentation uses 20-odd-letter case sensitive versions. So then you have to constantly cross reference example docs with the alias list if you want to have reasonable commands instead of a few paragraphs per input. And, yeah, I've run into strange issues with MinGW bash where commands throw mysterious errors beyond simply not being on my system.

Reply:
I hate bloat so will mark as resolved, but if you guys have any specific thing you want added please make a pull request on my branch :)